LLMonFHIR/Settings/Prompt.swift

```diff
@@ -45,7 +45,10 @@ enum Prompt: String {
     
     
     var prompt: String {
-        UserDefaults.standard.string(forKey: rawValue) ?? defaultPrompt
+        let preferredLanguage = Locale.preferredLanguages[0]
```

Comment:
You can probably directly move this in the string interpolation at line 50 as you don't really use this property somewhere else.

A more general comment: I would generally suggest to keep creating properties closer to their first use, e.g. in this case right above line 50 if you want to keep it a constant property. Makes the code easier to read as this is less of a context switch for the reader.